dataland-frontend/src/components/forms/parts/fields/MultiSelectFormField.vue

```diff
@@ -21,7 +26,40 @@ import { DropdownOptionFormFieldProps } from "@/components/forms/parts/fields/Fo
 
 export default defineComponent({
   name: "MultiSelectFormField",
+  emits: ["selectedValuesChanged"],
   components: { MultiSelectFormElement, UploadFormHeader },
-  props: DropdownOptionFormFieldProps,
+  data() {
+    return {
+      internalSelections: this.modelValue,
+    };
+  },
+  props: {
+    ...DropdownOptionFormFieldProps,
+    optionLabel: {
+      type: String,
+      default: "label",
+    },
+    optionValue: {
+      type: String,
+      default: "value",
+    },
+    modelValue: {
+      type: Array,
+      default: () => [],
+    },
+    ignore: {
+      type: Boolean,
+      default: false,
+    },
```

Comment:
These could be just the `MultiSelectFormProps`.
For this, please rename the current `MultiSelectFormElementProps` to `MultiSelectFormProps`, because it is used in both components then: in `MultiSelectFormField.vue` and `MultiSelectFormElement.vue`

Reply:
👍